java-generator/core/src/main/java/io/fabric8/java/generator/nodes/JEnum.java

```diff
@@ -47,20 +47,29 @@ public class JEnum extends AbstractJSONSchema2Pojo {
   private final String underlyingType;
   private final Set<String> values; //Let's prevent duplicates
 
-  public JEnum(String type, String underlyingType, List<JsonNode> values, Config config, String description,
```

Comment:
This is effectively, a breaking API change.
Can we preserve the original constructor, deprecate it, and have a safe fallback when pkg is `null`?